sdk/src/main/java/io/radar/sdk/RadarBeaconUtils.kt

```diff
@@ -0,0 +1,70 @@
+package io.radar.sdk
+
+import android.bluetooth.le.ScanFilter
+import android.bluetooth.le.ScanRecord
+import android.os.Build
+import android.util.Log
+import androidx.annotation.RequiresApi
+import io.radar.sdk.model.RadarBeacon
+import java.nio.ByteBuffer
+import java.util.*
+
+@RequiresApi(Build.VERSION_CODES.LOLLIPOP)
+internal object RadarBeaconUtils {
+
+    private const val MANUFACTURER_ID = 76
```

Comment:
what's 76?

Reply:
76 is the manufacturer ID for Apple (iBeacon). I'll add a bunch of comments in this class.